test/libsolidity/SolidityNatspecJSON.cpp

```diff
@@ -866,19 +876,23 @@ BOOST_AUTO_TEST_CASE(dev_multiline_return)
 		}
 	)";
 
-	char const* natspec = "{"
-	"\"methods\":{"
-	"    \"mul(uint256,uint256)\":{ \n"
-	"        \"details\": \"Multiplies a number by 7 and adds second parameter\",\n"
-	"        \"params\": {\n"
-	"            \"a\": \"Documentation for the first parameter starts here. Since it's a really complicated parameter we need 2 lines\",\n"
-	"            \"second\": \"Documentation for the second parameter\"\n"
-	"        },\n"
-	"        \"returns\": {\n"
-	"            \"d\": \"The result of the multiplication and cookies with nutella\",\n"
-	"        }\n"
-	"    }\n"
-	"}}";
+	char const* natspec = R"R({
+		"methods":
+		{
+			"mul(uint256,uint256)": {
```

Comment:
```suggestion
			"mul(uint256,uint256)":
			{
```
not sure if we impose strict style requirements on this, but seems like that's how the rest does it...